### PR TITLE
[besu] added gcp-storageclass

### PR DESCRIPTION
**Pull request by Roshan13046**
**Changelog**
- Add gcp-storageclass
- Update main.yml file to add an entry

 

**Reviewed by**
@suvajit-sarkar 
@sownak 

 

**Linked issue**
#1295

### CODE REVIEW

File: docs/source/operations/adding_new_org_besu.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ network:
     # Tls trust value
     tm_trust: "ca-or-tofu"                  # Options are: "whitelist", "ca-or-tofu", "ca", "tofu"
     ## File location for saving the genesis file should be provided.
-    genesis: "/home/user/blockchain-automation-framework/build/besu_genesis"   # Location where genesis file will be fetched
+    genesis: "/home/roshanpkraut/project/blockchain-automation-framework/build/besu_genesis"   # Location where genesis file will be fetched
```
Comment (sownak) [inline docs/source/operations/adding_new_org_besu.md:82]: don't commit personalised changes

File: platforms/hyperledger-besu/charts/node_tessera/templates/deployment.yaml [hunk 2]
```diff
@@ -239,6 +239,7 @@ spec:
 
           TESSERA_MAIN=com.quorum.tessera.launcher.Main;
           java -Xms128M -Xmx128M -cp $SQL_JAR:$TESSERA_JAR:. $TESSERA_MAIN --configfile ${DDIR}/tessera-config-with-hosts.json -o jdbc.password=$PASSWORD;
+          tail -f /dev/null
```
Comment (sownak) [inline platforms/hyperledger-besu/charts/node_tessera/templates/deployment.yaml:242]: this is not to be committed

File: platforms/hyperledger-besu/releases/dev/carrier-bes/reviewer_rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (sownak) [inline platforms/hyperledger-besu/releases/dev/carrier-bes/reviewer_rbac.yaml:1]: dont commit release files

File: platforms/shared/configuration/site.yaml [hunk 4]
```diff
@@ -5,8 +5,8 @@
   - hosts: all
     gather_facts: no
     no_log: "{{ no_ansible_log | default(false) }}"
-  - import_playbook: network-schema-validator.yaml  
-    when: reset is undefined or reset == 'false'       
+  # - import_playbook: network-schema-validator.yaml
```
Comment (sownak) [inline platforms/shared/configuration/site.yaml:8]: should not be commented

**Review by sownak — CHANGES_REQUESTED** [review]

**Comment by sownak** [comment]
Latest changes look good, only issue is that the DCO sign-off is missing, also, please squash all commits to a single commit. (that should solve the DCO issue)

**Comment by Roshan13046** [comment]
> Latest changes look good, only issue is that the DCO sign-off is missing, also, please squash all commits to a single commit. (that should solve the DCO issue)

I mistakenly made the wrong changes. Trying to fix it.

**Review by sownak — APPROVED** [review]

**Pull request merged**